### PR TITLE
docs(www): Managing Pull Requests 

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
Documentation on how to manage Pull Requests! 

<details><summary>Rendered View</summary>

localhost_8000_contributing_managing-pull-requests_

</details>

Fixes https://github.com/gatsbyjs/gatsby/issues/12094

### CODE REVIEW

File: docs/contributing/managing-pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: Managing Pull Requests
+---
+
+## Pull Requests and the Community
+
+Gatsby is an open source project. We have over 2,000 contributors and so much of what makes Gatsby great is contributed by folks like you!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: I'm not sure if this section is necessary... we should have some kind of intro though. Maybe replace this with a section describing who this doc is for and what you'll learn by reading it?

learnings could be something like - learn how to review changes being submitted to Gatsby or how your own contributions will be reviewed

File: docs/contributing/managing-pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+- Documentation
+- Code
+- Starters or Site Showcase
+- Blog posts
+
+When looking at a PR for the first time, it can help to read up on linked issues or RFCs (if there are any) to gain context on what the PR intends to add or fix.
+
+> 💡If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
+
+Each kind of PR requires a different set of checks from us before they are merged in!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
Each kind of PR requires a different set of checks from us before they are merged in.
```

File: docs/contributing/managing-pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+- Who can approve/merge?
+  - CODEOWNERS / core team
+
+## Gotchas
+
+- While it's nice to help out and fix merge conflicts or linting issues, you might not be able to push to the author's fork and that's okay!
+
+## Frequently Asked Questions
+
+None at the moment but if you have one, add it here!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
None at the moment but if you have one, add it here.
```

File: docs/contributing/managing-pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  - [ ] Add details about being invited after your first contribution and what that allows one to do
+  - [ ] Talk to the Core Team
+- Who can review?
+  - Everyone!
+- Who can approve/merge?
+  - CODEOWNERS / core team
+
+## Gotchas
+
+- While it's nice to help out and fix merge conflicts or linting issues, you might not be able to push to the author's fork and that's okay!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- While it's nice to help out and fix merge conflicts or linting issues, you might not be able to push to the author's fork and that's okay.
```

File: docs/contributing/managing-pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+
+### Code
+
+For PRs that add code (whether a feature or fix), we look for the following:
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out.
```
Comment (pieh) [inline docs/contributing/managing-pull-requests.md]: Would "Code readability" fit better (instead of Code Quality)?

And reason for that is the code is being read much more often than it's being written, so making sure it's easily digestible is big priority

Comment (DSchau) [reply docs/contributing/managing-pull-requests.md]: I don't think code readability is what we should strive for. I think the "Code Quality" label is good, but the description should be changed. How about:

> focus on reasonable changes that will likely make code maintenance, comprehension, or correctness improved or more obvious

(tweak as needed!)

File: docs/contributing/managing-pull-requests.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+> 💡 If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
+
+## Verifying a Pull Request
+
+### General Guidelines
+
+Some general things to verify in a pull request are:
+
+- Links ought to be relative
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Links ought to be relative instead of absolute when linking to docs (`/docs/some-reference` instead of `https://www.gatsbyjs.org/docs/some-reference`)
```

Not sure there would be another use case besides linking to docs, but I think a little example could help 🙂

File: docs/contributing/managing-pull-requests.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+Each kind of PR also requires a different set of specific checks from us before they are merged in!
+
+Let's go over them real quick.
+
+#### Documentation
+
+We typically look for the following in PRs that add documentation:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Style — whether the written language follows our [style guide](/contributing/gatsby-style-guide/)
```

Goes along with the advice to use relative links above

File: docs/contributing/managing-pull-requests.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+#### Code
+
+For PRs that add code (whether a feature or fix), we look for the following:
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests, and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
```

The style guide says use the Oxford comma so I guess I'll be nitpicky here 😬

File: docs/contributing/managing-pull-requests.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Code Quality — while it isn't good to nitpick, reasonable changes that improve readability are great to point out
```

Nitpick level 💯

File: docs/contributing/managing-pull-requests.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+- Links — check if the links are working and accessible
+- Tags — ensure the tags are not arbitrary
+- Featured Status — for the site showcase, we like to default to _not_ featuring a site and typically someone from the Gatsby team features it later if they like!
+
+#### Blog posts
+
+For PRs that add a blog posts, we ought to check:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Style — whether the written language follows our [style guide](/contributing/gatsby-style-guide/)
```

File: docs/contributing/managing-pull-requests.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+- Blog posts are similar to documentation in that they're long form writing, but also want to ensure that blog posts on [gatsbyjs.org](https://www.gatsbyjs.org) aren't purely promotional, spammy or inappropriate.
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds [gatsbyjs.org](https://www.gatsbyjs.org).
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]: I'm not really sure what the link to .org is here for?

File: docs/contributing/managing-pull-requests.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+> 💡 npm run format
+
+This will format all code and documentation and should get rid of those pesky linting issues!
+
+## Other Checks
+
+### Testing Locally
+
+While we have many _many_ tests in our repository (that are run automatically on every commit), there can be times when there exists an edge case (or five) that isn't covered by these.
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]: Edge case or five 😆 nothing to change here, just good touch haha

File: docs/contributing/managing-pull-requests.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+
+In situations like this, testing the change locally can be very valuable.
+
+> 💡 In case this is the first time you're doing this, you might have to [set up your development environment](https://www.gatsbyjs.org/contributing/setting-up-your-local-dev-environment).
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
> 💡 In case this is the first time you're doing this, you might have to [set up your development environment](/contributing/setting-up-your-local-dev-environment).
```

File: docs/contributing/managing-pull-requests.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+
+### Who can review a PR?
+
+If you're a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation on GitHub, you can review a PR.
+
+> 💡 Not a member yet? Love us and want to help? Make your first contribution and get invited!
+
+### Who can approve a PR?
+
+Every PR opened in the repository needs to be approved before it can be merged. While anyone who is a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation can approve a PR, to be merged in, it needs to be reviewed by a member of the team that owns that part of Gatsby.
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
Every PR opened in the repository needs to be approved before it can be merged. While anyone who is a member of the [gatsbyjs](http://github.com/gatsbyjs) organization can approve a PR, to be merged in, it needs to be reviewed by a member of the team that owns that part of Gatsby.
```

File: docs/contributing/managing-pull-requests.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+- Link to examples when necessary
+- Try not to [bikeshed](http://bikeshed.com/) too much
+- Note when a suggestion is optional (as opposed to required)
+- Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
+
+## Rights and Permissions
+
+### Who can review a PR?
+
+If you're a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation on GitHub, you can review a PR.
```
Comment (gillkyle) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
If you're a member of the [gatsbyjs](http://github.com/gatsbyjs) organization on GitHub, you can review a PR.
```

File: docs/contributing/managing-pull-requests.md [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+Simply put, when someone wants to contribute to Gatsby, they open a request to _pull_ their code into ours. Depending on the type of change, we like to categorise pull requests (or PRs for short) into:
+
+- Documentation
+- Code
+- Starters or Site Showcase
+- Blog posts
+
+When looking at a PR for the first time, it can help to read up on linked issues or RFCs (if there are any) to gain context on what the PR intends to add or fix.
+
+> 💡 If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: I don't think I agree with this. In many (most?) cases it's useful to open an issue before creating a PR.
 
But if there's an open PR that doesn't have a related issue, discussion typically happens on the PR itself. I can't think of an example where retroactively opening an issue would be useful.

File: docs/contributing/managing-pull-requests.md (hunk 2 above)
Comment (m-allanson) [reply docs/contributing/managing-pull-requests.md]: These are all outdated but still valid suggestions. You could find and replace all `!` for `.` and then add _one_ back in where it's most needed 😄 ❗️

File: docs/contributing/managing-pull-requests.md [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+Some general things to verify in a pull request are:
+
+- Links ought to be relative
+- Language ought to be inclusive and accessible
+
+### Type Specific Guidelines
+
+Each kind of PR also requires a different set of specific checks from us before they are merged in!
+
+Let's go over them real quick.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: Slightly too casual? Yes I'm a nitpick too :) 

```suggestion
Let's review them below.
```

File: docs/contributing/managing-pull-requests.md [hunk 18]
```diff
@@ -0,0 +1,178 @@
...
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out
+- Documentation in the package's README if you're adding something
+
+#### Starters or Site Showcase
+
+For PRs that add a site or a starter to the showcase, we ought to check:
+
+- We check if the site or starter is indeed a Gatsby site or starter respectively
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Check if the site or starter is built with Gatsby
```

File: docs/contributing/managing-pull-requests.md [hunk 19]
```diff
@@ -0,0 +1,178 @@
...
+- Links — check if the links are working and accessible
+- Tags — ensure the tags are not arbitrary
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Tags — ensure the tags match existing tags
```

File: docs/contributing/managing-pull-requests.md [hunk 20]
```diff
@@ -0,0 +1,178 @@
...
+- Featured Status — for the site showcase, we like to default to _not_ featuring a site and typically someone from the Gatsby team features it later if they like!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Featured Status — new sites should not be marked as featured. Featured sites are occasionally updated by a member of the Gatsby team 
```

File: docs/contributing/managing-pull-requests.md [hunk 21]
```diff
@@ -0,0 +1,178 @@
...
+
+#### Blog posts
+
+For PRs that add a blog posts, we ought to check:
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
For PRs that add a blog post, we ought to check:
```

File: docs/contributing/managing-pull-requests.md [hunk 22]
```diff
@@ -0,0 +1,178 @@
...
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+- Blog posts are similar to documentation in that they're long form writing, but also want to ensure that blog posts on [gatsbyjs.org](https://www.gatsbyjs.org) aren't purely promotional, spammy or inappropriate.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Subject matter - blog posts should not be purely promotional, spammy, or inappropriate. An author should check with a member of the Gatsby team that their post is appropriate for the Gatsby blog before creating their pull request.
```

File: docs/contributing/managing-pull-requests.md [hunk 23]
```diff
@@ -0,0 +1,178 @@
...
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds [gatsbyjs.org](https://www.gatsbyjs.org).
+
+We want all of these checks to pass. While it's okay to review a work in progress PR with some failed checks, a PR is only ready to ship when everything is green.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
We want all of these checks to pass. While it's okay to review a work in progress PR with some failed checks, a PR is only ready to ship when all the tests have passed.
```

File: docs/contributing/managing-pull-requests.md [hunk 24]
```diff
@@ -0,0 +1,178 @@
...
+
+Let's go over some common failure cases in our cases and how to approach fixing them:
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
Let's go over some common failure cases and how to fix them:
```

File: docs/contributing/managing-pull-requests.md [hunk 25]
```diff
@@ -0,0 +1,178 @@
...
+
+### Linting
+
+We lint all code and documentation pretty aggressively (consistency is good) and while that is great, you might find some PRs failing the linting check.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
We lint all code and documentation for consistency. You might find that your PR fails on the linting check.
```

File: docs/contributing/managing-pull-requests.md [hunk 26]
```diff
@@ -0,0 +1,178 @@
...
+
+An easy fix for this is to run:
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
If this is your PR and you have the code checked out on your machine, you can run:
```

File: docs/contributing/managing-pull-requests.md [hunk 27]
```diff
@@ -0,0 +1,178 @@
...
+
+> 💡 npm run format
+
+This will format all code and documentation and should get rid of those pesky linting issues!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
This will automatically re-format your changes to match the linting requirements. Don't forget to git commit and push your new changes.
```

File: docs/contributing/managing-pull-requests.md [hunk 28]
```diff
@@ -0,0 +1,178 @@
...
+
+Testing out unpublished packages locally can be tricky. We have just the tool to make that easy for you.
+
+Say hello to your new best friend, `gatsby-dev-cli`.
+
+#### gatsby-dev-cli
+
+`gatsby-dev-cli` is a command-line tool for local Gatsby development. When making changes in gatsby packages, this helps copy changes in the packages to a Gatsby site that you can test your changes on.
+
+Learn more about `gatsby-dev-cli` [here](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli)!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: Use descriptive link text

```suggestion
[Check the `gatsby-dev-cli` readme to learn more](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli).
```

File: docs/contributing/managing-pull-requests.md [hunk 29]
```diff
@@ -0,0 +1,178 @@
...
+
+### Commit and PR Title
+
+It's good to have descriptive commit messages so that other folks can make sense of what your commit is doing. We like [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3).
+
+However, PRs get squashed when merged into our repository so individual commit messages are not as important as PR titles.
+
+Let's look at some examples of good and bad PR titles:
+
+#### Good PR Titles ✅
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: Explain what's good about these titles?

File: docs/contributing/managing-pull-requests.md [hunk 30]
```diff
@@ -0,0 +1,178 @@
...
+
+- chore(docs): Fix links in contributing page
+- feat(gatsby): Add support for per page manifests
+- fix(gatsby-plugin-sharp): Ensure images exist before attempting conversion
+
+#### Bad PR Titles ❌
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: And what's bad about these ones.

File: docs/contributing/managing-pull-requests.md [hunk 31]
```diff
@@ -0,0 +1,178 @@
...
+- Note when a suggestion is optional (as opposed to required)
+- Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
+
+## Rights and Permissions
+
+### Who can review a PR?
+
+If you're a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation on GitHub, you can review a PR.
+
+> 💡 Not a member yet? Love us and want to help? Make your first contribution and get invited!
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: Ok this is exciting. Valid ❗️usage 😄 
```suggestion
> 💡 Not a member yet? Want to get involved in contributing to open source projects? Make your first contribution and you'll be invited automatically!
```

File: docs/contributing/managing-pull-requests.md [hunk 32]
```diff
@@ -0,0 +1,178 @@
...
+
+PRs can only be merged by members of the core team and Gatsbot.
+
+#### Gatsbot
+
+Gatsbot is our little android friend that automatically merges PRs that are ready to go. If a PR is approved and all checks are passing, add the `bot: merge on green` label and Gatsbot will merge it in automatically!
+
+## Gotchas
+
+- You might not always be able to push to an author's fork (depending on their settings). In such a case, leave suggestions.
```
Comment (m-allanson) [inline docs/contributing/managing-pull-requests.md]: Maybe explain this in a bit more detail or drop it completely?

Comment (m-allanson) [reply docs/contributing/managing-pull-requests.md]: Could drop the Gotchas section completely?

File: docs/contributing/managing-pull-requests.md (hunk 9 above)
Comment (marcysutton) [reply docs/contributing/managing-pull-requests.md]: You could mention Prettier does some of the work too...?

File: docs/contributing/managing-pull-requests.md (hunk 22 above)
Comment (marcysutton) [reply docs/contributing/managing-pull-requests.md]: One thing you could add is that blog posts are more time dependent than docs, especially since they get buried after more posts are published. If something is continually relevant and more of a general how-to, it should probably go in the docs "reference guides".

File: docs/contributing/managing-pull-requests.md [hunk 33]
```diff
@@ -0,0 +1,178 @@
...
+
+#### Bad PR Titles ❌
+
+- new tests
+- add support for my new cms
+- fix bug in gatsby
+
+## Giving Feedback
+
+- Be _nice_. We're stronger everyday because of our community so empathy is important and we want our contributors to feel welcome.
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
- Be _kind_. We're stronger every day because of our community, so compassion is important. We want all contributors to feel welcome.
```

File: docs/contributing/managing-pull-requests.md [hunk 34]
```diff
@@ -0,0 +1,178 @@
...
+- Make suggestions using [GitHub's suggestions feature](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) if possible. This makes accepting your suggestions easier for the author.
+- Link to examples when necessary
+- Try not to [bikeshed](http://bikeshed.com/) too much
+- Note when a suggestion is optional (as opposed to required)
+- Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]: Should we add anything about understanding when something is out of scope and best in a separate PR?

File: docs/contributing/managing-pull-requests.md (hunk 8 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: I love Oxford commas! Thank you

File: docs/contributing/managing-pull-requests.md (hunk 32 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: Updated with more detail!

File: docs/contributing/managing-pull-requests.md (hunk 11 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: I meant preview builds for .org but forgot to add the word _for_

Fixed!

File: docs/contributing/managing-pull-requests.md (hunk 22 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: Done and done!

File: docs/contributing/managing-pull-requests.md (hunk 17 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: But I like the casual style! 😛

File: docs/contributing/managing-pull-requests.md (hunk 16 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: This makes sense! Updating

File: docs/contributing/managing-pull-requests.md (hunk 2 above)
Comment (sidharthachatterjee) [reply docs/contributing/managing-pull-requests.md]: I see what you did there with the _one_ 😉

File: docs/contributing/managing-pull-requests.md [hunk 35]
```diff
@@ -0,0 +1,189 @@
+---
+title: Managing Pull Requests
+---
+
+## Introduction
+
+If you'd like to help out managing PRs on the gatsby repo on GitHub, this document is for you. We'll go over conventions we prefer, what we like to check for on various types of pull requests, permissions and guidelines on how to leave feedback.
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]: One thing I just noticed is that this doc deviates from the recommended "you" and "you're" style that we recommend in docs to focus on the reader. But in this instance, managing PRs is very much reliant on the core team's guidance and permissions, so I think it's fine. But something to keep in mind for future docs!

File: docs/contributing/managing-pull-requests.md [hunk 36]
```diff
@@ -0,0 +1,189 @@
...
+Each kind of PR also requires a different set of specific checks from us before they are merged in.
+
+Let's go over them real quick.
+
+#### Documentation
+
+We typically look for the following in PRs that add documentation:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](/contributing/gatsby-style-guide/)
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]: Something we have to correct a lot is the heading level. We could add a line here with:

`- Headings – whether the heading levels start with h2 (## in Markdown) and grow in order, establishing an accessible content hierarchy`

Comment (marcysutton) [reply docs/contributing/managing-pull-requests.md]: We could also point people to the contributing docs: 

`– Type & Format – whether docs and learning materials align with our recommendations for [docs contributions](https://www.gatsbyjs.org/contributing/docs-contributions/)

File: docs/contributing/managing-pull-requests.md [hunk 37]
```diff
@@ -0,0 +1,189 @@
...
+
+#### Code
+
+For PRs that add code (whether a feature or fix), we look for the following:
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]:
```suggestion
For [PRs that add code](/contributing/code-contributions/) (whether a feature or fix), we look for the following:
```

File: docs/contributing/managing-pull-requests.md [hunk 38]
```diff
@@ -0,0 +1,189 @@
...
+
+Let's go over some common failure cases and how to fix them:
+
+### Linting
+
+We lint all code and documentation for consistency. You might find that your PR fails on the linting check.
+
+If this is your PR and you have the code checked out on your machine, you can run:
+
+> 💡 npm run format
```
Comment (marcysutton) [inline docs/contributing/managing-pull-requests.md]: While fun, I wonder if the lightbulb will confuse people at all. Perhaps it should be marked up as a bash code snippet?

```bash
npm run format
```

File: docs/contributing/managing-pull-requests.md (hunk 17 above)
Comment (marcysutton) [reply docs/contributing/managing-pull-requests.md]: Since "quick" is relative, we could split the difference with `Let's go over them below.`

File: docs/contributing/managing-pull-requests.md (hunk 36 above)
Comment (marcysutton) [reply docs/contributing/managing-pull-requests.md]: I made these changes directly in the doc! 👍
